src/backend/libs/efiling-lookup-client/src/main/java/ca/bc/gov/open/jag/efilinglookupclient/ServiceFees.java

```diff
@@ -0,0 +1,106 @@
+package ca.bc.gov.open.jag.efilinglookupclient;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.joda.time.*;
+
+import java.math.BigDecimal;
+
+
+public class ServiceFees {
+
+    @SuppressWarnings("squid:S00107")
```

Comment:
do we need a comment here about why we're supressing this?

Reply:
Done!

Reply:
Thank you :)